lib/reporter/process.rb

```diff
@@ -47,6 +47,15 @@ def rss
 
     def collect_scheduler_stats(metric)
       metric.deferred_jobs_queued = Scheduler::Defer.length
+
+      metric.job_failures = {}
+
+      if Discourse.respond_to?(:job_exception_stats)
+        Discourse.job_exception_stats.each do |klass, count|
+          metric.job_failures[{ "job" => klass.to_s }] = count
+        end
+        metric.job_failures = failures
```

Comment:
```suggestion
```

O this line is causing the specs to fail.

Reply:
oops bad rebase :) removing.